Set the structure of the tutorials section (#538)
NixOS/nix.dev: source/tutorials/first-steps/index.md

# First Steps

> This tutorial series is meant to be your first interaction with Nix.

Reviewer: ```suggestion
This tutorial series is where you should start learning Nix.
```

Reviewer: This probably also needs some brief overview of what you'll learn in this series.
